benchmark/hits/prepare_data.sh

```diff
@@ -5,17 +5,17 @@ set -e
 current_dir=$(dirname "$0")
 
 q_start=$(date +%s.%N)
-if [[ ${HITS_DATA_URL} =~ "^http.*" ]]; then
+#if [[ ${HITS_DATA_URL} =~ "^http.*" ]]; then
     # Download benchmark target data
     # wget --continue https://datasets.clickhouse.com/hits_compatible/hits.parquet
-    wget --continue ${HITS_DATA_URL} -O /tmp/hits.parquet
-else
-    cp ${HITS_DATA_URL} /tmp/hits.parquet
-fi
+#    wget --continue ${HITS_DATA_URL} -O /tmp/hits.parquet
+#else
+#    cp ${HITS_DATA_URL} /tmp/hits.parquet
+#fi
 ${SQL_CLI} --file ${current_dir}/sql/create.sql >/dev/null
 q_end=$(date +%s.%N)
 q_time=$(echo "$q_end - $q_start" | bc -l)
 
-data_size=$(wc -c </tmp/hits.parquet)
+data_size=$(wc -c </data/hits.parquet)
```

Comment:
Please remind in README.md ` the hits data must be named hits.parquet under /data` and remove env `HITS_DATA_URL` if necessary

Reply:
ok